web/packages/shared/components/UnifiedResources/ListView/ResourceListItem.tsx

```diff
@@ -183,51 +184,51 @@ export function ResourceListItem({
           <ResTypeIconBox mr={1}>
             <SecondaryIcon size={18} />
           </ResTypeIconBox>
-          <HoverTooltip
-            tipContent={resourceType}
-            css={`
-              // Required for text-overflow: ellipsis to work. This is because a flex child won't shrink unless
-              // its min-width is explicitly set.
-              min-width: 0;
-            `}
-            showOnlyOnOverflow
-          >
-            <Text fontSize="14px" fontWeight={300} color="text.slightlyMuted">
+          <HoverTooltip tipContent={resourceType} showOnlyOnOverflow>
```

Comment:
I couldn't find a story to test this.

Reply:
I'm afraid there's no way to test this other than by manually editing HTML in dev tools. 
The grid is configured in a way that the column is always wide enough to fit the current resource types. But this may change in the future, if we introduce resources with longer types.